Dockerfile

```diff
@@ -18,19 +28,57 @@ COPY \
 # https://github.com/python-poetry/poetry-plugin-export/issues/210
 RUN pip install poetry==$POETRY_VERSION && \
     ./hack/build-wheels.sh /opt/mlserver/dist && \
+    _extras=() && \
+    if [[ "${OPT_MLSERVER_SYS_TRACING}" == "sdt-sys-tracing" ]]; then \
+        _extras+=("-E tracepoints"); \
+    fi && \
     poetry export --with all-runtimes \
         --without-hashes \
+        "${_extras[@]}" \
         --format constraints.txt \
         -o /opt/mlserver/dist/constraints.txt && \
     sed -i 's/\[.*\]//g' /opt/mlserver/dist/constraints.txt
 
+
+# Build native dependencies for tracepoints; 
+# Almalinux is binary-compatible with rhel ubi images but contains repositories
+# with additional devel packages (elfutils-libelf-devel needed here)
```

Comment:
Thanks for the note! I was just wondering about the need for `almalinux`!

Reply:
I'll also investigate if I can easily add one of the `almalinux` repositories to `ubi9/ubi-minimal` so that we can reuse the cached ubi-minimal image.

Reply:
Probably not worth it TBH - having tried it in the past, it tends to be a pain adding repos to UBI images (some of them are also behind "paywalls" and can only be used if you have a RHEL subscription).